frontend/app/src/composables/blockchain/index.ts

```diff
@@ -110,32 +111,112 @@ export const useBlockchains = () => {
       }
 
       if (supportsTransactions(chain)) {
-        await fetchDetected(chain, addresses);
+        await fetchDetected(chain, [address]);
       }
       await refreshAccounts(chain);
     };
 
     const promiseResult = await Promise.allSettled(
       payload.payload.map(async account => {
-        const addresses = await addEvmAccount(account);
-        if (isEmpty(addresses)) {
-          return notify({
-            title: t('actions.balances.blockchain_accounts_add.error.title', {
-              blockchain: 'EVM'
-            }),
+        const add = await addEvmAccount(account);
+        const { added, existed, failed, noActivity, ethContracts } = add;
+
+        const notificationTitle = t(
+          'actions.balances.blockchain_accounts_add.task.title',
+          {
+            blockchain
+          }
+        );
+
+        const getChainsText = (chains: string[], explanation?: string) =>
+          chains
+            .map(chain => {
+              let text = `- ${get(getChainName(chain))}`;
+              if (explanation) {
+                text += ` (${explanation})`;
+              }
+              return text;
+            })
+            .join('\n');
+
+        if (added) {
+          const [address, chains] = Object.entries(added)[0];
+
+          chains.forEach(chain => {
+            if (!isBlockchain(chain)) {
+              logger.error(`${chain} was not a valid blockchain`);
+              return;
+            }
+            startPromise(finishAddition(chain, address));
+          });
+
+          const isAll = chains.length === 0 && chains[0] === 'all';
+
+          notify({
+            title: notificationTitle,
             message: t(
-              'actions.balances.blockchain_accounts_add.error.empty_addresses_description',
-              { address: account.address }
+              'actions.balances.blockchain_accounts_add.success.description',
+              {
+                address: account.address,
+                list: !isAll ? getChainsText(chains) : ''
+              },
+              isAll ? 1 : 2
             ),
+            severity: Severity.INFO,
             display: true
           });
         }
-        for (const chain in addresses) {
-          if (!isBlockchain(chain)) {
-            logger.error(`${chain} was not a valid blockchain`);
-            continue;
-          }
-          startPromise(finishAddition(chain, addresses[chain]));
+
+        const listOfFailureText = [];
+        if (noActivity) {
+          listOfFailureText.push(
+            getChainsText(
+              Object.values(noActivity)[0],
```

Comment:
why do we only take the first value here?

Reply:
because we separate the tasks, so 1 task for 1 address. The response will show only 1 key there